src/entities/observation-entity.ts

```diff
@@ -280,10 +286,13 @@ export class Observation implements AbleToExportAndImportEuring {
   public remarks: string | null;
 
   // Not presented in euring standart
-  @IsOptional()
-  @IsBoolean()
-  @Column('boolean', { default: false })
-  public verified: boolean;
+  @IsEnum(Verified)
+  @Column({
+    type: 'enum',
+    enum: Verified,
+    default: Verified.Pending,
+  })
+  public verified: Verified;
```

Comment:
i think this should be optional, and filled by default with 'pending'

Reply:
done